backend/api/src/db/image/recent.rs

```diff
@@ -7,12 +7,12 @@ use shared::{
 use sqlx::PgPool;
 use uuid::Uuid;
 
-pub async fn create(
+pub async fn update(
```

Comment:
These three queries should be able to be cleaned up into a single query using postgres's `on conflict on constraint user_recent_image_user_id_image_id_key do ...` syntax.

See 
* https://www.postgresqltutorial.com/postgresql-upsert/
* https://stackoverflow.com/questions/17267417/how-to-upsert-merge-insert-on-duplicate-update-in-postgresql